data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,58 @@ mission "Remnant: Shattered Light 6"
 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	landing
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find and conduct an outfit scan of the Korath ship that can carry three 'olofez. The Remnant believe it will be found in one of the southern Korath systems."
+	source
+		government "Remnant"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	on offer
+		event "remnant: tubfalet appearance" 20 31
+		conversation
+			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
```

Comment:
After you say you can help. Chilia says "the ship should be somewhere in the nearer Korath systems." What ship? No ship is ever mention on this line. Perhaps add something about how they suspect that there is a single ship that carried those three 'olofezs there. That is, if it's a single ship with three bays then you know it's new.

Reply:
Expanded this along the lines of what you suggested.